### ISSUE TITLE
Text compare returns NaN

### ISSUE THREAD

**Issue by sayginburak** [toy/pHash#7]
I created two files; first.txt and second.txt 

And when I try to compare contents of this files 

```
Phash::Text.new('first.txt') % Phash::Text.new('second.txt')
```

the output is **NaN**

Is this not working anymore?

**Comment by toy** [comment]
After running `Phash::Text.new('Gemfile') % Phash::Text.new('Gemfile.lock')` in the project directory and checking code, I can say that this is a limitation of original library. Text hashes produced by the library have different length and for a very short `Gemfile` the length is 0 which produces **NaN** when compared to other file.
I've created this gem as an interface to the library, but to compare images and that part of the library seems to work fine (though at some point I've also created and mostly switched to `phamilie` gem which uses patched methods from pHash library, but does this only for images). Unfortunately the `pHash` library itself does not seem to be maintained as I've got no reply for patches I've sent there 2 years ago and there were no new releases.
